Revert #if DEBUG

rclee opened the pull request:
It's too dangerous to be kept alive!
![image](https://user-images.githubusercontent.com/10932745/75077194-e5898f00-54b6-11ea-8f93-e6089075900a.png)

rclee committed:
Revert #if DEBUG

alfwatt reviewed (approved):
![image](https://user-images.githubusercontent.com/53469/75077376-6779b800-54b7-11ea-97ae-d29521daa739.png)

Commit:
→ Add a Release testing target
→ Fix id/number mismatch in MMEConfiguration
→ Fix eventsCount warning when in Release mode
→ Fix Test
→ Move Service Fixture to port 8888

rclee committed:
Replace pushDebugEventWithAttributes: with macro

rclee committed:
Merge branch 'rclee-if-debug-revert' of https://github.com/mapbox/mapbox-events-ios into rclee-if-debug-revert
